en/guides/user-management/README.md

```diff
@@ -0,0 +1,127 @@
+
+![image](./staff.users_filtered_edit_link.en.png)
+
+## Management permissions
+
+In order to perform certain action on Dodona, you require additional **management permissions**. As a [teacher](#teacher), you can use the property <span class="guilabel">Permissions</span> on the [profile page](/en/for-students#profile-page) of a user to find out which permissions that user has.
+
+![image](./staff.user_edit_permission.en.png)
+
+## User permissions icon
+Users can be assigned the following [management permissions](#management-permissions):
+
+ | name                                          | icon                                |permissions|
+ | ----------------------------------------------|-----------------------------------------|----------|
+ | <span class="guilabel">Student</span>|   *no icon*|                                  [students](#student) can [edit](/en/for-students#user-profile-edit) their [user profile](/en/for-students#user-profile),[register](/en/for-students#course-register) for [courses](/en/course-management#course) and [submit](/en/for-students#submit-solution) [solutions](/en/for-students#solution) for [exercises](/en/for-students#exercise)|
+ |<span class="guilabel">Staff</span>|     ![image](../../../images/role_icons/staff.png)|   [teachers](#teacher) have all the permissions [students](#student) have and can additionally appoint teachers,, [edit](/en/for-students#user-profile-edit) the [profile](/en/for-students#user-profile) of students and teachers, [create](/en/course-management#create-a-course) [courses](/en/course-management#course) , [add](/en/course-management#add-exercise) [exercises](/en/for-students#exercise) and [add](/en/creating-a-judge) [judges](/en/for-students#judge) |
+ |<span class="guilabel">Zeus</span>|      ![image](../../../images/role_icons/zeus.png)|    [admins](#admin) have all the permissions [teachers](#teacher) and course administrators have and can additionally assign all [management permissions](#management-permissions) (including course managers), [edit](/en/for-students#edit-user-profile) the [profile](/en/for-students#user-profile) of all users, [edit](/en/course-management#edit-course) all [courses](/en/course-management#course), [edit](/en/course-management#edit-exercise) all [exercises](/en/for-students#exercise) and [edit]((/en/judges#edit-judge)) all [judges](/en/for-students#judge).
+
+### Student
+A user with <span class="guilabel">Student</span> [management permissions](#management-permissions)permissions is called a **student**.
+### Teacher
+A user with <span class="guilabel">Staff</span> [management permissions](#management-permissions)permissions is called a **teacher**.
+### Admin
+A user with <span class="guilabel">Zeus</span> [management permissions](#management-permissions)permissions is called an **admin**.
+
+## Assigning permissions
```

Comment:
Assigning site-wide permissions is also not possible anymore as a teacher.

They can however make enrolled members in their coruses _course admin_. That's done trough the 'make course admin' button in the users table of a course.